spec/features/admin/categories/platform_admin_can_visit_categories_index_page_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
this test could maybe be a little bit more robust--maybe actually adding multiple categories and checking to be sure that all categories are showing up on this page